cmd/podman/pods/logs.go

```diff
@@ -89,6 +89,9 @@ func logsFlags(cmd *cobra.Command) {
 
 	flags.BoolVarP(&logsPodOptions.Names, "names", "n", false, "Output container names instead of container IDs in the log")
 	flags.BoolVarP(&logsPodOptions.Timestamps, "timestamps", "t", false, "Output the timestamps in the log")
+
+	flags.BoolVarP(&logsPodOptions.DiffColors, "colored", "", false, "Output the different pods in different colors in the log")
```

Comment:
Obove the option is --colors?

Reply:
Why plural? I think just `--color` would be even better.

Reply:
could we make it a string so in future we can extend it if needed?

e.g. GNU diff and ls have `--color={never,always,auto}` where `--color=auto` detects if the stdout is a tty before turning on colors.

Reply:
We can also extend a bool in the future, it is easy to change it to a string while still keeping the `--color` only syntax.